src/lib/sql/sql_plan_cache.hpp

```diff
@@ -3,14 +3,14 @@
 #include <memory>
 #include <string>
 
-#include "cache/cache.hpp"
+#include "cache/gdfs_cache.hpp"
 
 namespace opossum {
 
 class AbstractOperator;
 class AbstractLQPNode;
 
-using SQLPhysicalPlanCache = Cache<std::shared_ptr<AbstractOperator>, std::string>;
-using SQLLogicalPlanCache = Cache<std::shared_ptr<AbstractLQPNode>, std::string>;
+using SQLPhysicalPlanCache = GDFSCache<std::string, std::shared_ptr<AbstractOperator>>;
+using SQLLogicalPlanCache = GDFSCache<std::string, std::shared_ptr<AbstractLQPNode>>;
```

Comment:
Having to explicitly talk about implementation details (GDFS), especially in the absence of alternatives, doesn't feel right here.

Reply:
Okay, so what would you suggest?

Reply:
No idea. Still wanted to point it out. :D

As it's the only place where this happens, I could live with it.